src/pages/components/button/style.mdx

```diff
@@ -237,12 +239,13 @@ There are three common button sizes: default, field, and small. Each button type
 can use any of these three sizes based on need. The fourth button size, full
 bleed, has a more reserved application and should rarely be used.
 
-| Button size | Height (px / rem) | Use case                                                                           |
-| ----------- | ----------------- | ---------------------------------------------------------------------------------- |
-| Full bleed  | 64 / 4            | Use when buttons bleed to the edge of a component, like in side panels and modals. |
-| Default     | 48 / 3            | Use as primary page actions and other standalone actions.                          |
-| Field       | 40 / 2.5          | Use when buttons are paired with input fields.                                     |
-| Small       | 32 / 2            | Use when there is not enough vertical space for the default sized button.          |
+| Button size      | Height (px / rem) | Use case                                                                           |
+| ---------------- | ----------------- | ---------------------------------------------------------------------------------- |
+| Full bleed       | 64 / 4            | Use when buttons bleed to the edge of a component, like in side panels and modals. |
+| Default          | 48 / 3            | Use as primary page actions and other standalone actions.                          |
+| Field            | 40 / 2.5          | Use when buttons are paired with input fields.                                     |
+| Small            | 32 / 2            | Use when there is not enough vertical space for the default sized button.          |
+| Large expressive | 48 / 3            | (TBD)                                                                              |
```

Comment:
We're not adding an additional size to the button since the default size is the same size used for expressive, so this can be removed.